neuromation/api/disks.py

```diff
@@ -0,0 +1,73 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, AsyncIterator, Mapping
+
+from yarl import URL
+
+from .config import Config
+from .core import _Core
+from .utils import NoPublicConstructor
+
+
+@dataclass(frozen=True)
+class Disk:
+    id: str
+    storage: int  # In bytes
+    owner: str
+    status: "Disk.Status"
+    cluster: str
```

Comment:
In all other classes (`_ConfigData`, `JobDescription`, `RemoteImage`, `_QuotaInfo`) we use attributes named `cluster_name`. Name `cluster` is never used. It is rather by accident, but it would be better to continue the tradition.